src/sentry/web/api.py

```diff
@@ -648,6 +648,13 @@ def post(self, request, project, event_id, project_config, **kwargs):
 
         project_id = project_config.project_id
 
+        event = eventstore.get_event_by_id(project_id, event_id)
```

Comment:
This will do a snuba query at every attachment upload.
Did you check with james if this is manageable in prod?

Sorry for missing it before.